src/traces/scattermapbox/convert.js

```diff
@@ -417,6 +417,10 @@ function getTextFont(trace) {
     else if(str === 'Open Sans Regular Bold Italic') str = 'Open Sans Bold Italic';
     else if(str === 'Klokantech Noto Sans Regular Italic') str = 'Klokantech Noto Sans Italic';
 
+    // Openmaptiles called it Extra Bold, instead of Extrabold - https://github.com/openmaptiles/fonts/issues/24
```

Comment:
Oh that's the issue I opened.
It looks like you are fixing a separate bug here?
If that's the case, it would be nice if you add a minimal PR for it and we merged it on master so that you use it here.

Reply:
Moved to
- #7020